### PR TITLE
Feat(827): update rdv sync

### PR DESCRIPTION
**Pull request by kevbarns**
- add historization script to remove training not found in the training catalogue

### CODE REVIEW

**Review by rimiti — APPROVED** [review]

File: server/src/jobs/rdv/controlAvailableFormationWithCatalogue.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ */
+export const controlAvailableFormationWithCatalogue = async () => {
+  logger.info("Cron #controlAvailableFormationWithCatalogue started.")
+
+  const stats = {
+    AncientElligibleTrainingCount: 0,
+    NewElligibleTrainingCount: 0,
+  }
+
+  stats.AncientElligibleTrainingCount = await WidgetParameter.countDocuments()
```
Comment (alanlr) [inline server/src/jobs/rdv/controlAvailableFormationWithCatalogue.ts]: ca marche countDocuments() sans ({}) ?

File: server/src/jobs/rdv/controlAvailableFormationWithCatalogue.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        if (!exist) {
+          await WidgetParameterHistoric.create({ ...formation, email_rdv: undefined, historization_date: dayjs().format() })
+          await WidgetParameter.findOneAndRemove({ cle_ministere_educatif: formation.cle_ministere_educatif })
+        }
+      },
+      { parallel: 500 }
+    )
+  )
+
+  stats.NewElligibleTrainingCount = await WidgetParameter.countDocuments()
```
Comment (alanlr) [inline server/src/jobs/rdv/controlAvailableFormationWithCatalogue.ts]: idem

File: server/src/jobs/rdv/controlAvailableFormationWithCatalogue.ts (hunk 1 above)
Comment (kevbarns) [reply server/src/jobs/rdv/controlAvailableFormationWithCatalogue.ts]: yes :-)

**Review by alanlr — APPROVED** [review]

**Pull request merged**